pkg/querier/tenantfederation/tenant_federation.go

```diff
@@ -0,0 +1,14 @@
+package tenantfederation
+
+import (
+	"flag"
+)
+
+type Config struct {
+	// Enabled switches on support for multi tenant query federation
+	Enabled bool `yaml:"enabled"`
+}
+
+func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
+	f.BoolVar(&cfg.Enabled, "tenant-federation.enabled", false, "If enabled, multi tenant query federation can be used by supplying multiple tenant IDs in the read path (experimental).")
```

Comment:
**suggestion(optional):** I think this help text can be reworked to explain how to use multi-tenant queries using `|` character in the X-Scope-OrgID.

**nit(optional,if-minor):** It might make sense to reword the phrase `multi tenant query federation can be used`. Changing it to something that describes query data from multiple tenants without using phrasing similar to the config field. Ideally, the help text should describe what a feature means to someone who can't parse the meaning from the config field name.

Reply:
@jtlisi Can you have another look. I tried to improve it according to both of your suggestions

Reply:
👍 Perfect!